testproject/urls.py

```diff
@@ -1,8 +1,15 @@
-from django.conf.urls import patterns, url
+import django
 from testproject import views
 
+if django.VERSION[1] < 9:
```

Comment:
This doesn't seem like a very robust way of checking for the version of Django - what happens when we get to 2.9? I think `django.VERSION < (1, 9)` might be better?

Reply:
Good point, fixed